core/views.py

```diff
@@ -15,13 +15,13 @@ def __init__(self):
         super(HashedNameServiceClient, self).__init__()
 
     def name_to_hash(self, name):
-        return requests.get(f"http://35.178.109.215:5000/name2hash/{urllib.parse.quote(name)}").text
+        return requests.get(f"https://cbp-id-provider.herokuapp.com/name2hash/{urllib.parse.quote(name)}").text
```

Comment:
I tested this using the prefilled name value [JOHNIE BAUMKIRCHNER](https://cbp-id-provider.herokuapp.com/name2hash/JOHNIE%20BAUMKIRCHNER).

👍